Gateway/Request/PaymentDataBuilder.php

```diff
@@ -47,7 +47,7 @@ public function build(array $buildSubject)
 
         $store_id = $order->getStoreId();
         $om = \Magento\Framework\App\ObjectManager::getInstance();
-        $manager = $om->get(Magento\Store\Model\StoreManagerInterface::class);
+        $manager = $om->get(\Magento\Store\Model\StoreManagerInterface::class);
```

Comment:
Out of curiosity, was this necessitated by the standards check?

Reply:
Yes, It was.